contracts/sui_x_oracle/pyth_rule/sources/pyth_adaptor.move

```diff
@@ -54,15 +33,15 @@ module pyth_rule::pyth_adaptor {
     // Make sure price confidence is within range
     assert_price_conf_within_range(price_value, price_conf);
     let price_decimals = (price_decimals as u8);
-    (price_value, price_conf, price_decimals, price_updated_time)
+    let now = clock::timestamp_ms(clock) /1000;
+    (price_value, price_conf, price_decimals, now)
   }
 
   fun assert_price_not_stale(price: &Price, clock: &Clock) {
-    // Check price time is within 60 seconds of current time
     let price_updated_time  = price::get_timestamp(price);
     let now = clock::timestamp_ms(clock) /1000;
-    assert!(price_updated_time <= now + 10, PYTH_PRICE_TOO_NEW);
```

Comment:
This condition won't happen, and it is not necessary to have this check. so we can safely remove this assert?